### ISSUE TITLE
Empty u.atoms.bonds for PDB files without a CONECT record

### ISSUE THREAD

**Issue by cbouy** [MDAnalysis/mdanalysis#2832]
## Expected behavior ##

For API consistency, when creating a Universe from a PDB file that doesn't have a CONECT record, accessing `u.atoms.bonds` should raise a `NoDataError` as it's the case for other file formats.

## Actual behavior ##

`u.atoms.bonds` will return a TopologyGroup with 0 bonds.

## Code to reproduce the behavior ##

``` python
>>> import MDAnalysis as mda
>>> from MDAnalysis.tests.datafiles import PDB_helix

>>> u = mda.Universe(PDB_helix)
>>> u.atoms.bonds
<TopologyGroup containing 0 bonds>

```

## Current version of MDAnalysis ##

- Which version are you using? (run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
2.0.0-dev0
- Which version of Python (`python -V`)?
Python 3.6.10
- Which operating system?
Ubuntu 18.04.4 LTS through a VM (WSL2)
